slides/week3/week3.org

```diff
@@ -2,66 +2,158 @@
 #+AUTHOR: Kristaps Berzinch, Raymond Ortiz
 #+EMAIL: kristaps@robojackets.org, rortiz9@gatech.edu
 
-* TODO Needs speaker notes
-
 * What are we doing today?
 - Data structures and algorithms in C++
 - Branching and merging in Git
+- Using light sensor to count grey squares
+
+* What is an object?
+- a module of reusable code
+  - maintains some sort of state
+    - variables
+  - have behaviors
+    - methods
+- can have multiple instances of the same object
```

Comment:
Did you mean class here? I guess that it's correct both ways though.

Reply:
I was trying to avoid saying classes since we have no covered them yet.